tests/test_wave_01.py

```diff
@@ -158,13 +158,14 @@ def test_moves_movie_from_watchlist_to_empty_watched():
     # Assert
     assert len(updated_data["watchlist"]) == 0
     assert len(updated_data["watched"]) == 1
-    
-    raise Exception("Test needs to be completed.")
+    assert MOVIE_TITLE_1 in janes_data["watched"][-1]['title']
```

Comment:
In this situation, since you're testing the value of strings, it might make sense to use `==` instead of `in`